sdk/storage/Azure.Storage.Blobs/src/Generated/BlobsModelFactory.cs

```diff
@@ -6,54 +6,11 @@
 #nullable disable
 
 using System;
-using System.Collections.Generic;
 
 namespace Azure.Storage.Blobs.Models
 {
     /// <summary> Model factory for models. </summary>
     public static partial class BlobsModelFactory
     {
```

Comment:
There is a custom `BlobsModelFactory` which already has factory method for `UserDelegationKey`. It was generated here because of the custom property handling in HLC